.github/workflows/add-new-pr-to-oss-triaging.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: Add external label to pull request if outside StackRox
```

Comment:
I can't comment on the lines, because they were not changed, so here goes the comment for `add-to-project` job.

I assume it is failing because of an issue with the `ADD_TO_PROJECT_TOKEN`. I asked around, but couldn't find the owner or contents of this secret.

Since the time we created the workflow, project automation feature was published by GH. I added a new workflow (https://github.com/orgs/stackrox/projects/2/workflows/17693587) to cover this job.
Therefore, this job can be removed from this workflow. The label still needs to be set by the `check-pr-if-external` job.